Project.toml

```diff
@@ -13,6 +13,7 @@ FunctionWrappers = "069b7b12-0de2-55c6-9aab-29f3d0a68a2e"
 Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
+NonlinearSolve = "8913a72c-1f9b-4ce2-8d82-65094dcecaec"
```

Comment:
This is a pretty heavy addition. Can the code that needs it be moved to an extension? For most jump problems this shouldn’t be needed.

Reply:
It should just be SimpleNonlinearSolve...